benchmark/hf3fs/bench_client.py

```diff
@@ -29,7 +29,7 @@ def test():
     file_size = 1 << 40
     bytes_per_page = 16 << 20
     entries = 32
-    file_ops = Hf3fsUsrBioClient(file_path, file_size, bytes_per_page, entries)
```

Comment:
I saw in `bench` there is also an initialization of `Hf3fsUsrBioClient`, do you want to change that as well?